common/src/main/java/whocraft/tardis_refined/common/tardis/manager/FlightDanceManager.java

```diff
@@ -107,10 +97,7 @@ private void updateControlsAfterDance() {
 
     @Override
     CompoundTag saveData(CompoundTag tag) {
-
-        // Save the position of the current console unit.
-
-        return null;
+        return tag;
```

Comment:
If ``saveData`` doesn't do anything, remove it

Reply:
They are required of from the BaseHandler object.